client/app/components/Chart/__tests__/ChartControl.spec.jsx

```diff
@@ -1,34 +1,31 @@
-/**
- * @flow
- * Enzyme samples: https://gist.github.com/richardscarrott/d89b37aff55ccc504193335198e676d1
- */
-import { shallow } from 'enzyme';
+// @flow
+
+import { render } from '@testing-library/react';
 import React from 'react';
 import { ChartControl } from '../ChartControl';
 
 describe('ChartControl', () => {
   it('renders', () => {
-    let wrapper = null;
-    expect(() => {
-      // shallow render only lists the first level of components
-      wrapper = shallow(
-        <ChartControl
-          types={['Moments']}
-          initialParams={{
-            type: 'Moments',
-            data: {
-              Moments: {
-                '2013-02-10 00:00:00 -0800': 11,
-                '2013-02-11 00:00:00 -0800': 6,
-              },
+    const { container, getByRole } = render(
+      <ChartControl
+        types={['Moments']}
+        initialParams={{
+          type: 'Moments',
+          data: {
+            Moments: {
+              '2013-02-10 00:00:00 -0800': 11,
+              '2013-02-11 00:00:00 -0800': 6,
             },
-          }}
-        />,
-      );
-    }).not.toThrow();
+          },
+        }}
+      />,
+    );
+
+    const chartControlBtn = getByRole('button');
+    const chart = getByRole('img');
 
-    expect(wrapper).not.toBeNull();
-    expect(wrapper.find('ChartControlButton').length).toEqual(1);
-    expect(wrapper.find('Chart').length).toEqual(1);
+    expect(container.firstChild).not.toBeNull();
```

Comment:
We can get rid of this expect since the subsequent imply that the component renders.